fixtures/cameo/root-par-6.json

```
@@ -0,0 +1,753 @@
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        }
+      ]
+    },
+    "Sound Sensitivity": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "SoundSensitivity",
+          "soundSensitivity": "off",
+          "comment": "triggers Strobe, Color Jumping & Fading"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "SoundSensitivity",
+          "soundSensitivityStart": "low",
+          "soundSensitivityEnd": "high",
+          "comment": "triggers Strobe, Color Jumping & Fading"
+        }
+      ]
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Dimmer 3": {
+      "name": "Dimmer",
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "1Hz",
+          "speedEnd": "20Hz"
+        }
+      ]
+    },
+    "Chase": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "Generic",
+          "comment": "No Chase, Color Preset 1 is on"
+        },
+        {
+          "dmxRange": [6, 26],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "durationStart": "10s",
+          "durationEnd": "4s",
+          "comment": " Chase of Ch 4 and Ch 5"
+        },
+        {
+          "dmxRange": [27, 127],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "durationStart": "4s",
+          "durationEnd": "0.1s",
+          "comment": " Chase of Ch 4 and Ch 5"
+        },
+        {
+          "dmxRange": [128, 148],
+          "type": "Effect",
+          "effectPreset": "ColorFade",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "durationStart": "10s",
+          "durationEnd": "4s",
+          "comment": " Chase of Ch 4 and Ch 5"
+        },
+        {
+          "dmxRange": [149, 255],
+          "type": "Effect",
+          "effectPreset": "ColorFade",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "durationStart": "4s",
+          "durationEnd": "0.1s"
+        }
+      ]
+    },
+    "Color Presets 1": {
```

Typo. Staying in-line with the DMX Table
```suggestion
    "Color Preset 1": {
```